doc/chips/11.rst

```diff
@@ -0,0 +1,183 @@
+Alternative Initializer Features
+================================
+
+Status:
+  Rejected
+
+Authors:
+  Kyle Brady, Lydia Duncan, Michael Ferguson, Mike Noakes, Vassily Litvinov (for Syntax 2);
+  Vassily Litvinov (for Child Fields and Initialization Completion)
+
+Abstract
+--------
+
+This document contains the features we discussed as part of the
+initializer story and chose not to incorporate.
+
+Rationale
+---------
+
+During our development of the initializer story for Chapel
+we considered a variety of features. Some of them were accepted
+and some of them were rejected. This document records some of
+the discussed-and-rejected features for posterity.
+
+Description
+-----------
+
+Alternative syntax for distinguishing Phase 1 and Phase 2 in an initializer
+***************************************************************************
+
+We considered providing dedicated syntax for distinguishing Phase 1
+and Phase 2 in an initializer. This syntax, codenamed Syntax 2,
+designates different block statements for Phase 1 and Phase 2,
+separated by the proposed "finalize" keyword.
+
+.. code-block:: chapel
+
+   proc init() {
+     ... Phase 1 code ...
+     super.init(...);
+   } finalize {
+     ... Phase 2 code ...
+   }
+
+Certain pieces of this syntax could be omitted as follows:
+
+* The super.init() call, if it takes no arguments.
+
+* The Phase 1 block statement, if contains no statements.
+  In this case, the "finalize" keyword can also be omitted.
+  Default field initializations followed by super.init() call
+  with no arguments occur in this case.
+
+Here is an example of an initializer where only Phase 2 is
+specified explicitly:
+
+.. code-block:: chapel
+
+   proc init() {
+     ... Phase 2 code ...
+   }
+
+Syntax 2 has the advantage of visually-clear separation of the two
+phases. We did not incorporate it because it did not allow sharing
+variables or extending an if statement across the two phases.
+
+Child's fields could depend on parent's fields
+**********************************************
+
+We considered allowing child's fields to be initialized based on
+parent's fields. Below is one way to achieve this.
+
+The child's initializer could control where, in its Phase 1, the
+parent's Phase 1 is invoked. In other words, the parent's Phase 1
+would not necessarily come at the end of the child initializer's Phase 1.
+The following example shows that based on Syntax 2.
```

Comment:
In this section you describe the strategy but not how it accomplishes allowing the child's fields to be initialized based on the parent's.  That mental leap does not follow easily from what you have described, so you should be more explicit.